recruiter/tests.py

```diff
@@ -32,7 +68,8 @@ def test_get_all_companies_returns_all_companies_as_a_query_set():
 
 @pytest.mark.django_db
 def test_get_all_recruiters_of_a_specified_company_returns_all_recruiters_as_a_query_set(example_recruiter):
-    recruiter_set = Company.get_all_recruiters_of_a_specified_company(Company.objects.get(name="example_company_a"))
```

Comment:
This reshaping of the code makes the PR bigger and makes it harder to find the code that needs to be reviewed. Please avoid that in the future.

Reply:
Ok